mozci/task.py

```diff
@@ -73,6 +74,7 @@ class Task:
     classification: str = field(default=None)
     classification_note: str = field(default=None)
     tags: Dict = field(default_factory=dict)
+    rev: str = field(default=None)
```

Comment:
I think instead of `rev` tasks should be associated with a `Push` instance since `rev` is not a unique identifier for a push (the same `rev` can appear on many different branches). A unique identifier would be branch + rev or possibly push id (though this involves querying hgmo to get).

Reply:
IIRC push ID isn't unique either